Calculator/Services/Delimiters/DelimiterService.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Calculator.Services.Delimiters
+{
+    public class DelimiterService : IDelimiterService
+    {
+        private const string DelimiterSeperator = "\n";
+        private const string CustomDelimiterIdentifierIndicator = "<";
+
+        public string[] GetDelimiters(string inputString, string delimiterIndicator)
+        {
+            var delimiters = new List<string> { ",", "\n" };
+
+            if (inputString.StartsWith(delimiterIndicator) || inputString.StartsWith(CustomDelimiterIdentifierIndicator))
```

Comment:
The only concern that I have here is that some functionality is "leaked" because we're sharing the delimiter service implementation - you could send an input string of "<(>)//(.)(,)\n1,2.3" and get 6 for the addition section, which isn't technically right.

Reply:
I initially had 2 implementations for the delmiter service but what you mention here is an oversight . Will fix that up .